unittests/test-contracts/test_api/test_action.cpp

```diff
@@ -1,15 +1,33 @@
-#include <eosiolib/action.hpp>
-#include <eosiolib/chain.h>
-#include <eosiolib/crypto.h>
-#include <eosiolib/datastream.hpp>
-#include <eosiolib/db.h>
-#include <eosiolib/eosio.hpp>
-#include <eosiolib/print.hpp>
-#include <eosiolib/privileged.h>
-#include <eosiolib/transaction.hpp>
+#include <eosiolib/contracts/eosio/action.hpp>
```

Comment:
The way the default includes work is that you can use #include <eosio/action.hpp>.  You don't need the eosiolib/[contracts|core|capi] part.  Also, we shouldn't be including the capi headers, use the c++ equivalent headers.  These will include the capi headers for you.